src/status_im2/contexts/quo_preview/profile/link_card.cljs

```diff
@@ -0,0 +1,125 @@
+                :customization-color :linkedin
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :mirror    {:title               "Mirror"
+                :icon                :social/mirror
+                :customization-color :mirror
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :opensea   {:title               "Opensea"
+                :icon                :social/opensea
+                :customization-color :opensea
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :pinterest {:title               "Pinterest"
+                :icon                :social/pinterest
+                :customization-color :pinterest
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :rarible   {:title               "Rarible"
+                :icon                :social/rarible
+                :customization-color :rarible
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :snapchat  {:title               "Snapchat"
+                :icon                :social/snapchat
+                :customization-color :snapchat
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :spotify   {:title               "Spotify"
+                :icon                :social/spotify
+                :customization-color :spotify
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :superrare {:title               "SuperRare"
+                :icon                :social/superrare
+                :customization-color :superrare
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :tumblr    {:title               "Tumblr"
+                :icon                :social/tumblr
+                :customization-color :tumblr
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :twitch    {:title               "Twitch"
+                :icon                :social/twitch
+                :customization-color :twitch
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :twitter   {:title               "Twitter"
+                :icon                :social/twitter
+                :customization-color :twitter
+                :on-press            #(js/alert "pressed")
+                :address             "@francescab"}
+    :youtube   {:title               "YouTube"
+                :icon                :social/youtube
+                :customization-color :youtube
+                :on-press            #(js/alert "pressed")
```

Comment:
consider removing the on-press from all of these entries an assoc it directly below?

Reply:
Done